src/ReinforcementLearningDatasets/Project.toml

```diff
@@ -22,24 +22,25 @@ ReinforcementLearningEnvironments = "25e41dd2-4622-11e9-1641-f1adca772921"
 Setfield = "efcf1570-3423-57d1-acb7-fd33fddbac46"
 TFRecord = "841416d8-1a6a-485a-b0fc-1328d0f53d5e"
 UnicodePlots = "b8865327-cd53-5732-bb35-84acbb429228"
+cuDNN = "02a925ec-e4fe-4b08-9a7e-0d78e3d38ccd"
 
 [compat]
 CodecZlib = "0.7"
 DataDeps = "0.7"
 Flux = "0.13, 0.14"
-HDF5 = "0.15"
+HDF5 = "0.15, 0.17"
 ImageCore = "0.9"
 NPZ = "0.4"
 PNGFiles = "0.3"
-Pickle = "0.2"
+Pickle = "0.3"
 Pipe = "1.3"
 ProgressMeter = "1.7"
 PyCall = "1.92"
-ReinforcementLearningBase = "0.9"
-ReinforcementLearningEnvironments = "0.6.3"
-Setfield = "0.7"
-TFRecord = "0.3"
-UnicodePlots = "1.3"
+ReinforcementLearningBase = "0.12"
```

Comment:
currently, rldatasets is unmaintained and not included in the ci/cd, please drop the changes to this file...